docs/dapp-staking/for-stakers/index.md

```diff
@@ -9,6 +9,9 @@ Please take the following parameters into consideration:
 - The **minimum** amount to stake for users is **50 SDN or 500 ASTR.**
 - **You need to claim to receive your rewards, we recommend claiming for your staking rewards once a week.**
 - There is a bonding period for around 5 days on Shiden and 10 days on Astar. Please note that this is based on a perfect block production of 12s. In case of any delay, your unbonding period can be a little longer.
+- The stake reward is shown in APY by default. APY is for stakers who have allowed compounding by turning on 'Re-stake after claiming". On the other hand, APR option is availble for stakers who haven't enabled compounding. The difference is that APR represents the annual rate simply for earning your stake but APY takes compounding into account. 
```

Comment:
Can you change some typo's?
The staking rewards are..
APR option is available for stakers...

Reply:
Fixed typos. Thanks.